cypress/component/Notes.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { mount } from '@cypress/react'
+import { Container, Notes } from '@toptal/picasso'
+import { TestingPicasso } from '@toptal/picasso/test-utils'
+
+const NotesTest = () => (
```

Comment:
We used to add `Example` postfix for Cypress component tests.

```suggestion
const NotesExample = () => (
```